stylesheet.css

```diff
@@ -1,4 +1,66 @@
 * {
-    margin: none;
-    padding: none
+    margin: 0;
```

Comment:
I am wondering if this doesn't duplicates the code? ¨margin: none¨ and ¨margin: 0¨? The same with padding.

Reply:
I think they are not duplicating. The red shows what has been removed and the green ones are added. So in this case I corrected the mistake - removed margin: none and padding: none and replaced them with margin: 0 and padding: 0

Reply:
Great! I didn't get it about red/green colors 😄